indico/migrations/versions/20220315_1520_5123f24eb41e_add_form_field_retention_period.py

```diff
@@ -0,0 +1,30 @@
+"""Add retention period to RegistrationFormItem
+
+Revision ID: 5123f24eb41e
+Revises: ef7a8b2e6737
+Create Date: 2022-03-15 15:20:59.230835
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '5123f24eb41e'
+down_revision = 'ef7a8b2e6737'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('form_items',
+                  sa.Column('retention_period', sa.Interval(), nullable=True),
+                  schema='event_registration')
+    op.add_column('form_items',
+                  sa.Column('is_purged', sa.Boolean(), nullable=False, server_default='false'),
```

Comment:
you need to remove the `server_default` again using `alter_column` after creating the column since we don't set this in the model